client/src/components/index.js

```diff
@@ -0,0 +1,6 @@
+import RefreshersView from "./RefreshersView";
+import Navbar from "./Navbar";
+import Navcrumbs from "./Navcrumbs";
+import RefreshersList from "./RefreshersList";
+
+export { RefreshersView, Navcrumbs, Navbar, RefreshersList };
```

Comment:
I really like this idea with the index file; It makes everything look super clean!